src/pages/Chat/steps/CreateOffer.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+import { SessionContainer } from '../../../containers/SessionContainer'
+import {
+  CreateOfferContainer,
+  getCreateOfferVariablesFromChatState,
+} from '../containers/CreateOfferContainer'
+import { OfferCreationSubscription } from '../containers/OfferCreationSubscription'
+import { mockState } from '../utils/test-utils'
+
+export const CreateOffer: React.SFC = () => (
+  <SessionContainer>
+    {(sessionToken) =>
+      !sessionToken ? (
+        'loading'
+      ) : (
+        <OfferCreationSubscription>
+          {(subscriptionMaybe) => (
+            <>
+              <CreateOfferContainer>
+                {(createOffer, { called }) => (
+                  <>
+                    <button
+                      onClick={() =>
+                        createOffer(
+                          getCreateOfferVariablesFromChatState(mockState()),
```

Comment:
Should this be left in? 🙂

Reply:
What happens if you double-click this button?

Reply:
lets disable it during loading/success state